update-orphaning/index.html

```diff
@@ -424,6 +430,117 @@
         }, 2000);
       }
 
+      // Populate version distribution pie chart
+      {
+        var orphanedVersions = data[4];
+        if (!orphanedVersions || !orphanedVersions["orphaned-by-versions"]) {
+          document.getElementById("version-distribution-chart-title").style.visibility = "hidden";
+          document.getElementById("version-distribution-chart").style.visibility = "hidden";
+        } else {
+          orphanedVersions = orphanedVersions["orphaned-by-versions"];
+          var versionDistributionObject = [];
+          var versionColors = {
+            "42": "#FF0000",
+            "43": "#FF8000",
+            "44": "#000099",
+            "45": "#FF00FF",
+            "46": "#99004C",
+            "47": "#A0A0A0",
+            "48": "#CCCC00",
+            "49": "#000066",
+            "50": "#0000FF",
+            "51": "#9933FF",
+            "52": "#009900",
+            "53": "#FFE5CC",
+            "54": "#663300",
+            "55": "#006666",
```

Comment:
This seems like something that could get out of hand, maintenance-wise.

Perhaps assign specific colours based on number of versions behind current?

If you're looking for colour-blind-friendly colours, you can check out the work I did for telemetry wrapper: https://github.com/mozilla/telemetry-dashboard/blob/gh-pages/wrapper/telemetry-wrapper.css#L85